Integrations/Netcraft/Netcraft.yml

```diff
@@ -6,19 +6,23 @@ display: Netcraft
 category: Authentication
 description: An integration for Netcraft, allowing you to open and handle takedown requests.
 configuration:
-- display: Use system proxy
-  name: proxy
+- display: Credentials
```

Comment:
```suggestion
- display: Username
```

Reply:
Why do you think we should use "Username" and not "Credentials"?

Reply:
have you ever seen a login which asked you for `Credentials` and not for `Username` ?

Reply:
But it's a username and password, not just username, wouldn't you agree?

Reply:
Password is there by default

Reply:
Fair enough, you're right